### ISSUE TITLE
Can't install from zip

### ISSUE THREAD

**Issue by Rahstyle** [primaeval/repository.primaeval#1]
I keep getting and error when trying to install the repository so I can get the add on. Thoughts? I tried commenting on the add on thread on the kodi site, but I'm login is broken.

**Comment by primaeval** [comment]
How did you add it? You have to download this zip and transfer it to your device. Make sure you have third party addons allowed in the settings.
https://github.com/primaeval/repository.primaeval/raw/master/zips/repository.primaeval/repository.primaeval-0.0.2.zip

**Comment by Rahstyle** [comment]
Ah so I can't download itike a standard add on in kodi? I have to side load it?

**Comment by primaeval** [comment]
Yes. Unless you put that zip on a web server that creates an index page. Github doesn't unfortunately. I've been looking for a free, non-spammy web host that would work but I haven't found anything yet.

**Comment by Rahstyle** [comment]
Cool, no worries thanks! OK so is the repository the only thing I need to side load then?

**Comment by primaeval** [comment]
I expect so. The IMDb repository can be installed from that repo if you need it. Most of my addons are in the main repo. I'm not sure which ones are broken right now. It is just beta quality. I think the BBC TV addon isn't working right now.

**Comment by Rahstyle** [comment]
I'm just trying to get the TV guide add on for my iptv:) can't stand the kodi guide:P

**Comment by primaeval** [comment]
The main repo should be all you need then. I'm glad you like it.

**Comment by Rahstyle** [comment]
If you have a chance, can you explain what I need to do to install it? :P when I go to install from zip, then USB, then the primaeval folder, then the Jarvis folder, it lists a whack of stuff and I'm not sure what to select:so lol

**Comment by Rahstyle** [comment]
Pfft nevermind, lol my scroll was stuck and it was literally the last one on the list lol

**Comment by Rahstyle** [comment]
OK so I got it installed. Do you have a link to setup instructions? I'm trying to get it working with vaders, but it can't load data. I probably missed something, but there's way more settings gs in here than PP or the standard kodi guide :P

**Comment by primaeval** [comment]
Most of it is explained in the wiki. Follow the QuickStart. It is a very advanced addon. You might not need something this powerful if you're using a simple source.
https://github.com/primaeval/script.tvguide.fullscreen/wiki

**Comment by Rahstyle** [comment]
OK thanks!

**Comment by Rahstyle** [comment]
Yeah you maybe right:/ I need to be able to enter the m3u for login and epg link for guide and I don't think I can add both
